tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -138,6 +138,51 @@ steps:
                   exit 1
                 fi
             else
+              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+
+              if [[ $release == "release" && $feedId == "build" ]]; then
+                packageVersion=$(grep -oP 'npm notice version:\s+\K[^\s]+' publish_log)
+                
+                if [ "${{ parameters.artifactPath }}" == "scoped" ]; then
+                  scopes=("fluidframework" "fluid-tools" "fluid-internal" "fluid-experimental")
+                  for scope in "${scopes[@]}"; do
+                      if [[ "$packageName" == "$prefix"* ]]; then
```

Comment:
Where is `$prefix` defined?

Reply:
OMG thanks for the catch! scope was previously named to prefix, I changed it to scope to make it clearer